AppBidding-MoPub/AppBidding-MoPub/BannerViewController.m

```diff
@@ -0,0 +1,43 @@
+//
+//  BannerViewController.m
+//  AppBidding-MoPub
+//
+//  Created by Abdoul Guisset on 6/29/20.
+//  Copyright © 2020 Abdoul. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "BannerViewController.h"
+#import "AdConfigurations.h"
+@import CriteoPublisherSdk;
+
+@interface BannerViewController ()
+
+@end
+
+@implementation BannerViewController
+
+
+// safeAreaInsets will be 0 if executed in viewDidLoad
+- (void)viewDidAppear:(BOOL)animated {
+
+    // Do any additional setup after loading the view.
+    self.adView = [[MPAdView alloc] initWithAdUnitId:[AdConfigurations mopubBannerAdUnitId]];
+    self.adView.delegate = self;
+    self.adView.frame = CGRectMake(0, self.view.frame.size.height-self.view.safeAreaInsets.bottom-50, self.view.frame.size.width, 50);
```

Comment:
Could you add spaces around the - operator ? Without it is hard to read the expression

Reply:
Done